Ginger/GingerCore/Drivers/Mobile/Perfecto/PerfectoDriver.cs

```diff
@@ -177,6 +182,19 @@ private DesiredCapabilities GetWebCapabilities()
 
             capabilities.SetCapability("deviceName", Perfecto_Device_ID);
             capabilities.SetPerfectoLabExecutionId(Perfecto_Host_URL);
+
+            if (!string.IsNullOrEmpty(ProxySettings))
+            {
+                Proxy p = new Proxy();
```

Comment:
why to have Proxy object creation code duplicated, please set it in function and call it for 2 methods

Reply:
Done